code/kthoom.js

```diff
@@ -692,6 +692,8 @@ class KthoomApp {
 
   showNextPage() {
     const turnedPage = this.bookViewer_.showNextPage();
+    if (this.bookViewer_.getFitMode() === FitMode.Width)
```

Comment:
Actually this seems like a good change regardless of the fit mode?

However, if you really only want to do this for FitMode.Width, can you pls enclose this if-block with curly braces?

Reply:
Yes of course I will do the update